### PR TITLE
No address information on a printed form letter

### PR DESCRIPTION
**Pull request by billyhunt**
[Link to ZenHub issue.](https://github.com/usdoj-crt/crt-portal-management/issues/1133)

## What does this change?

Names and Addresses currently display "None" in empty fields on printed form letter.  This change will remove the word "None" and leave it blank for any field.  

**note** when testing, make sure to try different combinations of address fields to make sure the fields with data still populate.

## Screenshots (for front-end PR):

# Previous
image
image

# After update
image
image

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for, document, and establish a testing plan for any behavior that may vary across environments or is otherwise difficult to test.
+ [x] Check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for any behavior that may vary across environments or is difficult to test, and ensure that it is well-understood, documented, and that there is a testing plan in place.
+ [x] Re-check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

**Comment by billyhunt** [comment]
Updated to show email address.

image

File: crt_portal/cts_forms/templates/forms/complaint_view/print/form_letter.html [hunk 1]
```diff
@@ -29,6 +29,10 @@ <h2 class="form-letterhead--header-subtext">Civil Rights Division</h2>
       {% endif %}
       <br />
       {% if data.contact_city %}{{ data.contact_city }},{% endif %} {% if data.contact_state %}{{ data.contact_state }}{% endif %} {% if data.contact_zip %}{{ data.contact_zip }}{% endif %}
+      {% if data.contact_email %}
```
Comment (louh) [inline crt_portal/cts_forms/templates/forms/complaint_view/print/form_letter.html:32]: Should `data.contact_email` also be added to the list of conditionals in line 19?

**Review by louh — CHANGES_REQUESTED** [review]
I haven't usually commented on git hygiene on this project but I think it's most warranted here, can we rewrite the history so that "Initial commit" doesn't get merged in?

Alternatively, we can resolve this by squash-merging or rebase-merging this branch, but that means one of us would need to remember to select that option during the merge.

**Comment by billyhunt** [comment]
>I haven't usually commented on git hygiene on this project but I think it's most warranted here, can we rewrite the history >so that "Initial commit" doesn't get merged in?

I am OK with that in theory, but have never done it. What does that process entail?

**Review by louh — APPROVED** [review]

**Pull request merged**